src/components/pages/Login/Login.js

```diff
@@ -6,6 +6,7 @@ import {
   loginHandler,
   getToken
 } from '../../../utils/handlers/authenticationHandlers';
+import PropTypes from 'prop-types';
 
 function Login(props) {
```

Comment:
Small thing: when using propTypes, it is generally considered best practice to destructure incoming props. It not of the highest priority and can on later PRs, but it's something to keep in mind :)

Reply:
That's true, just changed it :)